mobile-app/app/screens/AppNavigator/screens/Dex/components/PoolPairCards/PoolPairCards.tsx

```diff
@@ -116,17 +116,29 @@ export function PoolPairCards ({
 
   useEffect(() => {
     setIsSearching(false)
-    setFilteredYourPairs(
-      yourPairs.filter((pair) =>
-        pair.data.displaySymbol
-          .toLowerCase()
-          .includes(debouncedSearchTerm.trim().toLowerCase())
-      ).sort((a, b) =>
-        availablePairs.findIndex(x => x.data.id === a.data.id) -
-        availablePairs.findIndex(x => x.data.id === b.data.id
+    if (showSearchInput) {
```

Comment:
same here, can we do an early return to avoid nested if-condition?

Reply:
yes